tests/func/test_pull.py

```diff
@@ -154,14 +154,6 @@ def remote_dataset_info(requests_mock, remote_dataset):
     requests_mock.get(f"{STUDIO_URL}/api/datachain/datasets/info", json=remote_dataset)
 
 
-@pytest.fixture
-def remote_dataset_stats(requests_mock):
-    requests_mock.get(
-        f"{STUDIO_URL}/api/datachain/datasets/stats",
```

Comment:
do we use stats for anything else outside?

Reply:
only calls were in this file, I think we should remove the route from Studio